src/unix/lwt_unix.ml

```diff
@@ -659,6 +659,114 @@ let write_string ch buf pos len =
   let buf = Bytes.unsafe_of_string buf in
   write ch buf pos len
 
+module IO_vectors =
+struct
+  type _bigarray =
+    (char, Bigarray.int8_unsigned_elt, Bigarray.c_layout) Bigarray.Array1.t
+
+  type _buffer =
+    | Bytes of Bytes.t
+    | Bigarray of _bigarray
```

Comment:
I typically use the `_` to indicate at a glance that something isn't exposed (or in rare cases, like `_bigarray`, is exposed, but I wish it weren't).